CodeSniffer/Standards/Generic/Tests/Formatting/DisallowLongArraySyntaxUnitTest.fail.inc

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Fixture filename must match used sniff name or otherwise it might not be discovered by test runner.

Correct name would be `DisallowLongArraySyntaxUnitTest.inc`. And then fixed filename would be `DisallowLongArraySyntaxUnitTest.inc.fixed`.